docs/application/dotnet/get-started/wearable-connection/index.md

```diff
@@ -1,4 +1,4 @@
-# Test Tizen .NET Application on a Gear device
+# Connect Your Wearable
```

Comment:
Looks like old version of images are used, kindly ensure the latest version of images are copied for the following:
testing_your_app_on_gear1.png
testing_your_app_on_gear2.png
testing_your_app_on_gear3.png
testing_your_app_on_gear4.png
testing_your_app_on_gear5.png
testing_your_app_on_gear6.png
testing_your_app_on_gear7.png

Reply:
Images has been updated.